example/test.js

```diff
@@ -0,0 +1,9464 @@
+// source: protos/test.proto
+/**
+ * @fileoverview
+ * @enhanceable
+ * @suppress {missingRequire} reports error on implicit type usages.
+ * @suppress {messageConventions} JS Compiler reports an error if a variable or
+ *     field starts with 'MSG_' and isn't a translatable message.
+ * @public
+ */
+// GENERATED CODE -- DO NOT EDIT!
+/* eslint-disable */
+// @ts-nocheck
+
+import * as jspb from 'google-protobuf';
+var goog = jspb;
+var proto = {};
```

Comment:
This should go away in favor of imports (already done) and we should update all the types in the JSDoc strings to refer to the types we import.